settings.json.sample

```diff
@@ -0,0 +1,67 @@
+{
+    "server":{
+        "hostname":"test",
+        "scheme":"https",
+        "port":"",
+        "ssh.key_private":"/home/whoami/.ssh/id_hudson_dsa",
+        "ssh.username":"root"
+    },
+    "ldap":{
+        "ldaphost":"test",
+        "ldapuser":"test",
+        "ldappwd":"test",
+        "basedn":"test",
+        "grpbasedn":"test"
+    },
+    "robottelo":{
+        "project":"sat",
+        "locale":"en_US.UTF-8",
+        "remote":"0",
+        "smoke":"0",
+        "upstream":"1",
+        "verbosity":"2",
+        "screenshots.base_path":"/tmp/robottelo/screenshots/",
+        "virtual_display":"0",
+        "window_manager_command":"test"
+    },
+    "tests":{
```

Comment:
As a way to say to robottelo if some feature is avaiable or not, we can check if some key is defined. If manifest is present then that feature is "enabled".